src/NServiceBus.Gateway.AcceptanceTests/When_listening_on_wildcard_uri.cs

```diff
@@ -25,7 +25,30 @@ public EndpointWithWildCardUriAsDefault()
                 {
                     var gatewaySettings = c.Gateway();
 
-                    gatewaySettings.AddReceiveChannel("http://+:25699/");
+                    gatewaySettings.AddReceiveChannel("http://+:25699/WildcardA/");
+                });
+            }
+        }
+
+        [Test]
+        public void Should_not_throw_exception_if_wildcard_channel_is_not_default()
+        {
+            Assert.DoesNotThrowAsync(async () =>
+                await Scenario.Define<ScenarioContext>()
+                    .WithEndpoint<EndpointWithWildCardUriAndFullyQualifiedDefault>(e => e.When(b => Task.FromResult(0)))
+                    .Run());
+        }
+
+        class EndpointWithWildCardUriAndFullyQualifiedDefault : EndpointConfigurationBuilder
```

Comment:
do we need a dedicated configuration builder instead of just configuring it like in the test above? https://github.com/Particular/NServiceBus.Gateway/pull/99/files#diff-41766bade9040337cbed61a2aec26894R28

Reply:
Not following?

Reply:
I was thinking about using a single template for both tests and configure their channels as part of their `WithEndpoint<>(e => e.WithConfig(..)` config. But probably not that useful anyway.